modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1368,6 +1368,38 @@ For example, when there are three servers, each hosting databases `foo` and `bar
 **Scenario 2:** The cluster appears unbalanced, but server constraints prevent you from moving to a better, more balanced, allocation. For example, assuming server 1 hosts databases `foo` and `bar`, server 2 hosts only `foo`, and server 3 hosts no databases. Then, a better allocation would move `foo` from server 1 to server 3, but if server 3 has the constraint `deniedDatabases:['foo']}`, then the cluster is already balanced subject to this constraint.
 ====
 
+[#_neo_clientnotification_cluster_requestedtopologymatchedcurrenttopology]
+=== RequestedTopologyMatchedCurrentTopology
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.RequestedTopologyMatchedCurrentTopology
+|Title
+a| `<command>` has no effect.
```

Comment:
Following the pattern of other notifications here, e.g., `ServerAlreadyEnabled`. But should this say `ALTER DATABASE` has no effect here?

Reply:
If this will only ever happen for ALTER DATABASE we could hardcode it yes rather than keeping it dynamic. But either is fine with me